assembl/static2/js/app/graphql/QuestionQuery.graphql

```diff
@@ -0,0 +1,16 @@
+query QuestionQuery($lang: String!, $id: ID!) {
```

Comment:
Can you please just name it Question? (don't change the filename). This is because the generated types will be named QuestionQueryQuery otherwise, not great. :)

Reply:
done